common/bazaar.cpp

```
@@ -56,6 +56,31 @@ int16 Bazaar::ResolvePurchaseItemCharges(
+	SharedDatabase &db,
+	const EQ::ItemData *item,
+	uint32 quantity
+)
+{
+	std::vector<std::unique_ptr<EQ::ItemInstance>> items;
+	if (!item || quantity == 0) {
+		return items;
+	}
+
+	items.reserve(quantity);
+	for (uint32 i = 0; i < quantity; ++i) {
```

Clang (14.0.6)

[warning] variable 'i' is not initialized

(cppcoreguidelines-init-variables)